### DISCUSSION TITLE
Adding dark mode toggle component to TopNavigation

### DISCUSSION THREAD

**Discussion by nbaillie** [cloudscape-design/components#586] [Q&A] [answered]
Hi i was looking to see if there is any guidance on the way to add a dark mode toggle component to the TopNavigation 
(or inline with it).

Simalar to what is implemented here: https://cloudscape.design/

**Comment by timogasda** [comment, marked as answer]
Hello!

Unfortunately we currently do not have any official guidance for a dark mode toggle in general. So you can decide what type of UX makes the most sense for your application. For example, if you already have a place where users define their preferences, this could be a good place to add it.
